### PR TITLE
Logviewer streams

### PR DESCRIPTION
**Pull request by fabianfrz**
this will add the streams to the log viewer the same way as the ones for HTTP

### CODE REVIEW

**Comment by lennyev** [comment]
go
@fabianfrz
www/nginx: add logviewer support for stream

**Comment by fabianfrz** [comment]
@mimugmail the only problem is that the plugin has absolutely no UX optimization. See #1018 
If you can reorder the menus (reaonable for an user), the plugin could be even better.

File: www/nginx/src/opnsense/www/js/nginx/webpack.conf.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ module.exports = {
     },
     output: {
         path: path.resolve(__dirname, 'dist'),
-        filename: '[name].js'
+        filename: '[name].min.js'
```
Comment (fichtner) [inline www/nginx/src/opnsense/www/js/nginx/webpack.conf.js]: thanks for this

File: www/nginx/src/opnsense/mvc/app/controllers/OPNsense/Nginx/Api/LogsController.php [hunk 2]
```diff
@@ -57,10 +57,42 @@ public function errorsAction($uuid = null)
...
+        } else {
+            // emulate REST call for a specific log /stream_errors/uuid
+            return $this->call_configd_stream('streamerror', $uuid);
+        }
+    }
 
     private function call_configd($type, $uuid)
     {
-        if (!$this->vhost_exists($uuid)) {
+        if (!($this->vhost_exists($uuid))) {
```
Comment (fichtner) [inline www/nginx/src/opnsense/mvc/app/controllers/OPNsense/Nginx/Api/LogsController.php]: why the extra () ?

Comment (fabianfrz) [reply www/nginx/src/opnsense/mvc/app/controllers/OPNsense/Nginx/Api/LogsController.php]: not needed, I'll remove it before merge

**Pull request merged**